gcc 10 compilation error: multiple definition

1. With `gcc 10` on my `debian sid` , make can not compile successfully!  
```
make
```
```
gcc -O2   -o discoal discoal_multipop.c discoalFunctions.c ranlibComplete.c alleleTraj.c -lm
/usr/bin/ld: /tmp/ccwFmeUS.o:(.bss+0x40d1e4): multiple definition of `npops'; /tmp/ccnerq3Q.o:(.bss+0x40d204): first defined here
..
/usr/bin/ld: /tmp/ccwFmeUS.o:(.bss+0x40d1fc): multiple definition of `sampleNumber'; /tmp/ccnerq3Q.o:(.bss+0x40d21c): first defined here
collect2: error: ld returned 1 exit status
make: *** [Makefile:12: discoal] Error 1
```

2. It seems gcc 10 use `-fno-common` by default. Adding `-fcommon` flag makes it work again.
```
gcc -O2   -o discoal discoal_multipop.c discoalFunctions.c ranlibComplete.c alleleTraj.c -lm -fcommon
```
fixed this.

--- a/Makefile
+++ b/Makefile
@@ -9,7 +9,7 @@ all: discoal
 
 
 discoal: discoal_multipop.c discoalFunctions.c discoal.h discoalFunctions.h
-	$(CC) $(CFLAGS)  -o discoal discoal_multipop.c discoalFunctions.c ranlibComplete.c alleleTraj.c -lm
+	$(CC) $(CFLAGS)  -o discoal discoal_multipop.c discoalFunctions.c ranlibComplete.c alleleTraj.c -lm -fcommon
 	
 test: alleleTrajTest.c alleleTraj.c alleleTraj.h discoalFunctions.c
 	$(CC) $(CFLAGS)  -o alleleTrajTest alleleTrajTest.c alleleTraj.c ranlibComplete.c discoalFunctions.c -lm

